CONTRIBUTING.md

```diff
@@ -1,225 +1,245 @@
-# Contributing to WSO2 Thunder ⚡
+# **Contributing to Thunder**
 
-Thank you for your interest in contributing to WSO2 Thunder! This guide will help you set up your development environment and understand the contribution process.
+Thank you for your interest in contributing to Thunder! We welcome contributions of all kinds—from fixing typos to proposing major features. This guide will help you understand how you can contribute.
 
-## 📋 Table of Contents
+---
 
-- [Prerequisites](#prerequisites)
-  - [Required Tools (Core Development)](#required-tools-core-development)
-  - [Required Tools (Frontend Development)](#required-tools-frontend-development)
-- [Development Setup](#development-setup)
+## **🎯 Ways to Contribute**
 
-## Prerequisites
+Thunder welcomes different types of contributions, and you can participate at any level:
 
-Before you begin, ensure you have the following installed on your system:
+| Contribution Type | Documentation |
+| ----- | ----- |
+| **Bug Reports** | [Report a bug](#-reporting-bugs) |
+| **Documentation** | [Improve docs](#-improving-documentation) |
+| **Bug Fixes** | [Fix a bug](#-fixing-bugs) |
+| **Minor Improvements** | [Make improvements](#-making-improvements) |
+| **Share Ideas & Define Problems** | [Problem definition](#-sharing-ideas--defining-problems) |
+| **Design Features** | [Design proposals](#-designing-solutions) |
+| **Implement Features** | [Implementation guide](docs/contributing/development.md) |
 
-### Required Tools (Core Development)
+**You don't need to do everything!** You can:
 
-- **[Git](https://git-scm.com/downloads)** - Version control system
-- **[Go](https://golang.org/doc/install)** - Version 1.25 or higher
-- **[Node.js](https://nodejs.org/en/download/)** - Version 22 or higher (`LTS` is recommended ✅)
-- **[pnpm](https://pnpm.io/installation)** - Version 9 or higher (`LTS` is recommended ✅)
+* Share an idea and let someone else design/implement it
+* Define a problem and let someone else design the solution
```

Comment:
Lets remove this point? Even though it has a difference compared to first point, that difference might not get by a person who comes to this project for the first time and confuse with what exactly the difference.